app/assets/stylesheets/_colours.scss

```diff
@@ -0,0 +1,78 @@
+$black:         #000;
```

Comment:
@tobyhede the compiled *CSS* or *SCSS*?

- CSS: https://gov-au-ui-kit.apps.staging.digital.gov.au/latest/ui-kit.min.css
- SCSS: https://gov-au-ui-kit.apps.staging.digital.gov.au/latest/_ui-kit.scss